models/garden.py

```diff
@@ -46,8 +45,29 @@ def has_orchard(self) -> bool:
             bool: if garden has orchard
         """
 
+    # pylint: disable=line-too-long
+    def get_dict(self, data_type: Type) -> dict:
```

Comment:
not sure, but I think you can just use built-in "type" type, not necessary to use "Type" type. But you better try this